src/main/java/net/fabricmc/loom/util/srg/SpecialSourceExecutor.java

```diff
@@ -34,17 +44,6 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import com.google.common.base.Stopwatch;
-import org.apache.commons.io.output.NullOutputStream;
-import org.gradle.api.Project;
-import org.gradle.api.logging.LogLevel;
-import org.gradle.api.logging.configuration.ShowStacktrace;
-
-import net.fabricmc.loom.LoomGradleExtension;
-import net.fabricmc.loom.configuration.providers.forge.McpConfigProvider.RemapAction;
-import net.fabricmc.loom.util.FileSystemUtil;
-import net.fabricmc.loom.util.ThreadingUtils;
-
 public class SpecialSourceExecutor {
```

Comment:
This file is hard to review, did any functionality here change? Or is it a refactor?

Reply:
refactor, look between it and the old one and it's pulling the stripping stuff out to another function so I can use it separately